lib/migration-chunks/validation/duplicate-props.js

```diff
@@ -0,0 +1,52 @@
+const errorMessages = require('./errors');
+
+const isSameEntity = (step, otherStep) => {
+  return (
+    step.payload.contentTypeId === otherStep.payload.contentTypeId &&
+    step.payload.fieldId === otherStep.payload.fieldId
+  );
+};
+
+const hasProps = (step) => step.payload && step.payload.props;
```

Comment:
By our convention (based on the FSA) there should always be a payload - just maybe not as an object. So maybe a `typeof step.payload === 'object'` would make it clearer, that we only want actions in the format of `{ payload: { foo: 'bar' } }`